lm_eval/models/huggingface.py

```diff
@@ -358,6 +358,11 @@ def eot_token_id(self):
         # we use EOT because end of *text* is more accurate for what we're doing than end of *sentence*
         return self.tokenizer.eos_token_id
 
+    @property
+    def bos_or_eos_token_id(self):
+        # it used as prefix for rolling loglikelihood
```

Comment:
as seen in model.py, it is also used in loglikelihood (without rolling), so this comment increases confusion